pkg/portal/portal.go

```diff
@@ -406,7 +401,20 @@ func (p *portal) serve(path string) func(w http.ResponseWriter, r *http.Request)
 	}
 }
 
+func (p *portal) getResourceID(subscriptionID, resourceGroup, clusterName string) string {
+	return strings.ToLower(
+		fmt.Sprintf(
+			"/subscriptions/%s/resourceGroups/%s/providers/Microsoft.RedHatOpenShift/openShiftClusters/%s",
+			subscriptionID, resourceGroup, clusterName))
+}
+
 func (p *portal) internalServerError(w http.ResponseWriter, err error) {
 	p.log.Warn(err)
 	http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
 }
+
+func (p *portal) badRequest(w http.ResponseWriter, err error) {
+	p.log.Warn(err)
+	p.log.Debug(err)
```

Comment:
Last little nitpick but do we need both warn and debug here?

Reply:
ohh sorry, it should be only Debug